XRTK-Core/Packages/com.xrtk.core/Utilities/Editor/MixedRealityEditorSettings.cs

```diff
@@ -17,7 +17,7 @@ namespace XRTK.Utilities.Editor
     public class MixedRealityEditorSettings : IActiveBuildTargetChanged
     {
         private const string IgnoreKey = "_MixedRealityToolkit_Editor_IgnoreSettingsPrompts";
-        private const string SessionKey = "_MixedRealityToolkit_Editor_ShownSettingsPrompts";
+        public const string SessionKey = "_MixedRealityToolkit_Editor_ShownSettingsPrompts";
```

Comment:
Instead of making this public I should update this to be a simple flag property.